lib/Doctrine/Persistence/Mapping/RuntimeReflectionService.php

```diff
@@ -62,6 +62,8 @@ public function getClassNamespace($class)
 
     /**
      * {@inheritDoc}
```

Comment:
Are you sure this works properly? I have a vague memory if reading that inheritDoc was ignored by Psalm when there was other tags in the block

Reply:
You're right, I thought it read both since I got some other error about instantiating `ReflectionClass`, I've added the other tags.